inc/namespace.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace HM\Platform\CMS;
+
+use function HM\Platform\get_config;
+
+function bootstrap() {
+	require_once __DIR__ . '/remove_updates/namespace.php';
+	require_once __DIR__ . '/branding/namespace.php';
+
+	$config = get_config()['modules']['cms'];
+	Remove_Updates\bootstrap();
+
+	if ( $config['branding'] ) {
+		Branding\bootstrap();
+	}
+
+	if ( $config['login-logo'] ) {
+		add_action( 'login_header', __NAMESPACE__ . '\\add_login_logo' );
+	}
+}
+
+/**
+ * Add the custom login logo to the login page.
+ *
+ * @return void
```

Comment:
I wouldn't bother with this, but if you do, be consistent and use it on all the other functions too.